### ISSUE TITLE
Need a new version in NPM

### ISSUE THREAD

**Issue by alansouzati** [reactjs/react-magic#27]
Hi folks,

  Some improvements have been approved on this project that I really need in my app. Is that possible for you guys to release a new version in NPM?

Much appreciated.

Alan

**Comment by Daniel15** [comment]
I'd love to, but NPM seems broken right now. It's not even letting me log in:

```
$ npm login
Username:
Username: daniel15
Password:
Email: (npm@dan.cx)
npm http PUT https://registry.npmjs.org/-/user/org.couchdb.user:daniel15
npm http 400 https://registry.npmjs.org/-/user/org.couchdb.user:daniel15
npm ERR! Error: Unknown error while authenticating : -/user/org.couchdb.user:daniel15
```

Once I figure out if this is an issue on my end or an issue on NPM's end, I'll push an update.

**Comment by alansouzati** [comment]
Thanks Danilo, enjoy the rest of your evening!

On Mon, May 4, 2015 at 9:45 PM, Daniel Lo Nigro <notifications@github.com>
wrote:

> I'd love to, but NPM seems broken right now. It's not even letting me log
> in:
>
> $ npm login
> Username:
> Username: daniel15
> Password:
> Email: (npm@dan.cx)
> npm http PUT https://registry.npmjs.org/-/user/org.couchdb.user:daniel15
> npm http 400 https://registry.npmjs.org/-/user/org.couchdb.user:daniel15
> npm ERR! Error: Unknown error while authenticating : -/user/org.couchdb.user:daniel15
>
> Once I figure out if this is an issue on my end or an issue on NPM's end,
> I'll push an update.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reactjs/react-magic/issues/27#issuecomment-98947853>.
>



-- 
--
Alan Souza
Software Engineer
Contact: +1 (408) 421 - 6341

**Comment by Daniel15** [comment]
Posted to the NPM Github project to see if they know why I'm having issues logging in: https://github.com/npm/npm/issues/8180

**Comment by alansouzati** [comment]
Did you try updating NPM to see if you can publish? In the issue you've opened I was able to see the full stacktrace and consequently your NPM version. I know that this is probably only a work around, but it could solve the problem for now. It seems to be an authentication issue, so my inclination is to believe that is not related with versioning, but who knows.

**Comment by Daniel15** [comment]
Somehow I fixed it, 0.2.2 published: https://www.npmjs.com/package/htmltojsx

Thanks!

**Comment by alansouzati** [comment]
thanks!
